roles/common/tasks/ntp.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install NTP
+  yum:
+    name: ntp
+    state: present
+    update_cache: true
+
+- name: Configure NTP
+  template:
+    src: ntp.conf
+    dest: /etc/ntp.conf
+    owner: root
+    group: root
+    mode: 0644
+    backup: true
+  notify:
+    - restart ntp
+    - start ntp
```

Comment:
It is better to move `start ntp` handler here as a task step.

Reply:
Also, please add the timezone configuration step (see https://docs.ansible.com/ansible/2.9/modules/timezone_module.html)

Reply:
fixed (add timezone + move 'start ntp' handler to ntp role)